File: Python/counter.py
import random
import time

times_reached = 0
max_objectives = int(input("Enter the maximum number of objectives to reach: "))
while True:
    objective_time = random.randint(1, 10)
    counter = 0

    print(f"New Objective: {objective_time} seconds")

    while counter < objective_time:
        print(f"Counter: {counter} seconds")
        time.sleep(1)
        counter += 1

    times_reached += 1
    print(f"Objective time reached, {times_reached} times\n")
    if times_reached >= max_objectives:
        print(f"Objective time reached, {max_objectives} times")
        break

File: Java/counter.java
package DOKO_CODE;
import java.util.Scanner;
import java.util.Random;
import java.util.InputMismatchException;

public class counter { 
    public static void main(String[] args) throws InterruptedException {
        Random random = new Random();
        Scanner scanner = new Scanner(System.in);
        int timesReached = 0; 

        System.out.print("Enter the number of times to reach the objective: ");
        int maxObjectives = 0;

        
        try {
            maxObjectives = scanner.nextInt();
        } catch (InputMismatchException e) {
            System.out.println("Error: Please enter a valid integer.");
            scanner.close();
            return; 
        }

        while (true) {
            int objectiveTime = random.nextInt(10) + 1;
            int counter = 1; 

            System.out.println("\nNew Objective: " + objectiveTime + " seconds");

            
            while (counter <= objectiveTime) {
                System.out.println("Counter: " + counter + " seconds");
                Thread.sleep(1000);
                counter++;
            }
            
            timesReached++;
            System.out.println("Objective Reached " + timesReached + " times");
            
            if (timesReached >= maxObjectives) {
                System.out.println("Objective time reached, " + maxObjectives + " times");
                break;
            }
        }
        scanner.close();
    }
}